### ISSUE TITLE
Print to HTML to support previewer modifiers like vscode-markdown-mermaid

### ISSUE THREAD

**Issue by DamianSuess** [yzhang-gh/vscode-markdown#447]
#### What is the problem?
As a user, I'd like the Print to HTML to be able to support outputting Markdown Previewer modifier such as ``Markdown Preview Mermaid Support``. Currently, it 

#### How can I reproduce it?
1. Install ``vscode-markdown``
2. Install [vscode-markdown-mermaid](https://github.com/mjbvz/vscode-markdown-mermaid)
3. Create a simple markdown doc and add the following to make a ``sequence diagram``.
```mermaid
sequenceDiagram
  Alice->>Bob: Hello Bob, how are you?
  alt is sick
    Bob->>Alice: Not so good :(
  else is well
    Bob->>Alice: Feeling fresh like a daisy
  end
  opt Extra response
    Bob->>Alice: Thanks for asking
  end
```
4. Use vscode-markdown's ``Print current document to HTML`` feature to generate an HTML file
5. View formatted document

#### Is there any error message in the console?
Currently, the raw code is displayed and not the generated image

**Comment by 1354092549** [comment]
In discussion
#658

**Comment by yzhang-gh** [comment]
Thanks to @1354092549, it is resolved with v3.0.0
